hazelcast/src/hazelcast/client/config.cpp

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 #include <atomic>
+#include <boost/algorithm/string/case_conv.hpp>
```

Comment:
these changes are not relevant to config.cpp file, please revert `config.cpp` changes.